### DISCUSSION TITLE
RPM support?

### DISCUSSION THREAD

**Discussion by Destroyinator69420** [Tribler/tribler#7247] [General]
Hello Tribler team, I am using Fedora as my preferred linux distribution now. I am wondering when you will support RPM? Also, when will you make available an arm64 and armhf build for the raspberry pi and asus tinker board?

All the best,
Destroyer

**Comment by drew2a** [comment]
@Destroyinator69420 hi!

Unfortunately, we do not have any plans to support RPM as well as support arm64 and armhf builds in near future.

Here are the useful links (in case you try to do it yourself):

Debian build: https://github.com/Tribler/tribler/blob/main/build/debian/makedist_debian.sh
Docker build: https://github.com/Tribler/tribler/blob/main/.github/workflows/docker-image.yml
Ubuntu GitHub Action build: https://github.com/Tribler/tribler/blob/main/.github/workflows/build_ubuntu.yml
